Exercise_A/index.html

```diff
@@ -1,45 +1,190 @@
+          <div class="card-img-overlay text-center bg-dark rounded-0">
+            <h5 class="card-title">Two Thing</h5>
+            <p class="card-text">At</p>
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card bg-dark text-white">
+          <img class="card-img rounded-0" src="assets/images/3.jpg" alt="Card image">
+          <div class="card-img-overlay text-center bg-dark rounded-0">
+            <h5 class="card-title">Three Thing</h5>
+            <p class="card-text">This</p>
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card bg-dark text-white">
+          <img class="card-img rounded-0" src="assets/images/4.jpg" alt="Card image">
+          <div class="card-img-overlay text-center bg-dark rounded-0">
+            <h5 class="card-title">Four Thing</h5>
+            <p class="card-text">Bullshit</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div id="template" class="text-center">
+      <div class="text-center p-5">
+        <h2 class="text-light w-50 mx-auto p-2">Seriously, It's This Fucking Template!</h2>
+        <button type="button" class="wow animate__animated animate__bounce btn btn-light rounded-pill">THAT'S AMAZING</button>
+      </div>
+    </div>
+    <div class="row p-5 d-inline-flex justify-content-center">
+      <div class="text-center p-5 col">
+        <h2 class="text-dark w-50 mx-auto p-2">My Contact Information</h2>
+        <hr class="mx-auto">
+        <p class="text-dark  w-50 mx-auto p-2">Here's my phone number and email address, because hey that's part of the template and why would you not include that on your webpage? You'd be a fool not to! I can't wait for all the job offers to come pouring in from all the companies wanting my insane website design skills.
+         <br><br>Honestly this template does look really nice, though.</p>
+      </div>
+      <div class="d-inline-flex align-items-stretch flex-wrap justify-content-center">
+        <div class="col"></div>
+        <div class="col">
+          <div class="card text-center">
+            <i class="fas fa-thumbs-up fa-2x p-2"></i>
+            <div class="card-body">
+              <h5 class="card-title">+1-608-317-1888</h5>
+            </div>
+          </div>
+        </div>
+        <div class="col">
+          <div class="card text-center">
+            <i class="fas fa-heart fa-2x p-2"></i>
+            <div class="card-body">
+              <a><h5 class="card-title">holy@shit.com</h5></a>
+            </div>
+          </div>
+        </div>
+        <div class="col"></div>
+      </div>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.6.0.js" integrity="sha256-H+K7U5CnXl1h5ywQfKtSj8PCmoN9aaq30gDh27Xc0jk=" crossorigin="anonymous"></script>
+    <script src="assets/js/script.js"></script>
+    <script src="js/wow.min.js"></script>
+    <script>
+    new WOW().init();
+    </script>
+</body>

```

Comment:
Kelvin fantastic work, you have developed a very close clone to the original website!
You have most of the functionalities implemented properly and you have used bootstrap utilities very well to achieve the spacing on you page. 
With that in mind you just need to link a few of you buttons to the correct sections and just making some buttons do things.

Otherwise you have done a fantastic job well done and keep it up!